testing/jest/button.tests.tsx

```diff
@@ -116,6 +116,15 @@ describe('Button', () => {
             });
         });
 
+        describe('icon', () => {
+            it('should render icon', () => {
+                const button = render({ icon: 'test' });
+
+                expect(button.is('.dx-button-has-icon')).toBeTruthy();
+                expect(button.exists('.dx-icon.dx-icon-test')).toBeTruthy();
```

Comment:
`button.exists('.dx-icon.dx-icon-test')`
->
`button.find('.dx-icon.dx-icon-test').exists()`

Reply:
why?

Reply:
My fail. I thought that `exists` doesn't have any arguments.